uwblueprint/ccbc
# Make Preview Review Carousel Responsive

## Notion ticket link

[Make Preview Review Carousel Responsive](https://www.notion.so/uwblueprintexecs/Make-Preview-Review-Carousel-Responsive-4233f807966340e68cb564a642fc0274)



## Implementation description
Essentially just updating state to see if the current screen is a mobile view and if it is, render the carousel's preview information below the carousel to make it responsive.



## Steps to test
1. Look at a preview of a book
2. Change the screen width to see that the preview is dynamic and responsive
3. Look at how the carousel looks on mobile views


## What should reviewers focus on?
* 
Implementation details and if there is a better way to go about it.

## Checklist
